inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/task/FormatDateLogFileTask.java

```diff
@@ -17,11 +17,11 @@
 
 package org.apache.inlong.agent.plugin.task;
 
-import org.apache.inlong.agent.plugin.task.file.LogFileTask;
+import org.apache.inlong.agent.plugin.task.logcollection.local.FileTask;
 
 /**
  * Directory trigger with format date.
  */
-public class FormatDateLogFileTask extends LogFileTask {
+public class FormatDateLogFileTask extends FileTask {
```

Comment:
Is `FormatDateLogFileTask` really needed?

Reply:
It's useless, I'll delete it